packages/database/package.json

```
@@ -7,11 +7,13 @@
   "browser": "dist/index.esm.js",
   "module": "dist/index.esm.js",
   "esm2017": "dist/index.esm2017.js",
-  "files": ["dist"],
+  "files": ["dist", "exp/package.json", "exp-types"],
```

I don't think we want to include exp artifacts when publishing our prod package. If there is any special handling needed for the exp release, we will handle that in the exp release script similar to how firestore is handled.
```suggestion
  "files": ["dist"],
```